exporter/awsxrayexporter/translator/aws.go

```diff
@@ -156,39 +110,39 @@ func makeAws(attributes map[string]string, resource pdata.Resource) (map[string]
 	// progress from least specific to most specific origin so most specific ends up as origin
 	// as per X-Ray docs
 	if hostID != "" {
-		ec2 = &EC2Metadata{
-			InstanceID:       hostID,
-			AvailabilityZone: zone,
-			InstanceSize:     hostType,
-			AmiID:            amiID,
+		ec2 = &awsxray.EC2Metadata{
+			InstanceID:       aws.String(hostID),
```

Comment:
Alternatively, is it a big deal to just use `&` here instead of the utility?

Reply:
lol. It honestly hasn't even crossed my mind. Silly me! Will attempt to simplify it in a separate PR.